jetson/percep/pcl.hpp

```diff
@@ -52,25 +52,27 @@ class PCL {
         shared_ptr<pcl::visualization::PCLVisualizer> viewer;
```

Comment:
The indentation of these two lines doesn't look consistent with the rest of the file. Could you fix that?